### DISCUSSION TITLE
search engine optimination

### DISCUSSION THREAD

**Discussion by mrinal002** [bitmagnet-io/bitmagnet#363] [Ideas]
add a [Elasticsearch](https://www.elastic.co/elasticsearch) search engine optimination

  Seeders and Leechers update button
  automatic delete   Seeders : 0 and Leechers : 0

**Comment by rraymondgh** [comment]
I would expect as a minimum dependencies need to be self hosted. I've been experimenting with mongodb atlas self hosted for caching / finding content in classifier.   This is quite a heavy thing to self host (1.46GB docker image).  Intending to try meilisearch instead.   Self hosted or SaaS search engines have similar types of integration challenges.  Keeping master database (Postgres) and search caches in sync.  bitmagnet already uses Postgres full text search (FTS).   This does work pretty well.  Adding "fuzzy" search only finds about 5% additional matches after being filtered for decent similarity
